docs/pages/build-reference/how-tos.md

```diff
@@ -60,15 +60,9 @@ This is an example of how your package.json might look like:
 ## How to use private package repositories
 
 - Configure your project in a way that works with `yarn` and relies on the `NPM_TOKEN` env variable to authenticate with private repositories
-- add `experimental.npmToken` in `credentials.json`
+- add `NPM_TOKEN` to your account or project's secrets. See the [secret environment variables](/build-reference/variables/#using-secrets-in-environment-variables) docs to learn how to do this.
 
-```json
-{
-  "experimental": {
-    "npmToken": "example npm token"
-  }
-}
-```
+![Secret creation UI filled](/static/images/eas-build/environment-secrets/secrets-create-filled.png)
```

Comment:
let's use the [ImageSpotlight component](https://github.com/expo/expo/blob/master/docs/components/plugins/ImageSpotlight.tsx) here, eg: https://github.com/expo/expo/blame/da519ecf657b768f667d4eb3761ffc7b478b1def/docs/pages/build-reference/build-configuration.md#L15

original pr: https://github.com/expo/expo/pull/12106